star/codec_test.go

```diff
@@ -130,7 +130,7 @@ func TestEncode(t *testing.T) {
 	})
 
 	t.Run("go-string", func(t *testing.T) {
-		// go interfaces are not supported
+		// go workflow are not supported
```

Comment:
The comment `go interfaces are not supported` sounds fine. Was this change (interfaces -> workflow) due to accidental replace all? :)